crates/edr_napi/src/solidity_tests/test_results.rs

```diff
@@ -580,10 +580,19 @@ impl CallTrace {
     /// Instantiates a `CallTrace` with the details from a node and the supplied
     /// children.
     fn new(node: &traces::CallTraceNode, children: Vec<Either<CallTrace, LogTrace>>) -> Self {
-        let contract = node.trace.decoded.label.clone();
+        let contract = node
+            .trace
+            .decoded
+            .as_ref()
+            .and_then(|decoded| decoded.label.clone());
```

Comment:
`decoded` became an `Option<Box<_>>` to save memory in Foundry's REVM inspector.

@agostbiro I'm not sure whether the generation of `decoded` is guaranteed in our setup. Do you know?

If so, we could also use `expect()`.

Reply:
Thanks!